Source/Charts/Data/Implementations/Standard/ChartDataEntry.swift

```diff
@@ -108,32 +99,18 @@ open class ChartDataEntry: ChartDataEntryBase
     }
 }
 
-public func ==(lhs: ChartDataEntry, rhs: ChartDataEntry) -> Bool
-{
-    if lhs === rhs
-    {
-        return true
-    }
-    
-    if !lhs.isKind(of: type(of: rhs))
+// MARK: Equatable
+extension ChartDataEntry/*: Equatable*/ {
+    public static func ==(lhs: ChartDataEntry, rhs: ChartDataEntry) -> Bool
     {
-        return false
-    }
-    
-    if lhs.data !== rhs.data && !lhs.data!.isEqual(rhs.data)
-    {
-        return false
-    }
-    
-    if fabs(lhs.x - rhs.x) > Double.ulpOfOne
-    {
-        return false
-    }
-    
-    if fabs(lhs.y - rhs.y) > Double.ulpOfOne
-    {
-        return false
+        if lhs === rhs
+        {
+            return true
+        }
+
+        return lhs.isKind(of: type(of: rhs))
+            && (lhs.data?.isEqual(rhs.data) ?? true)
```

Comment:
To address your other point, there are a few areas where Charts has some issues (e.g. large data sets, performance with real time data, some API oddities). It is also clearly not a Swift project. I understand the language has evolved quite a bit since it's original release, but there are some key areas (e.g. why do `ChartData` and `ChartDataSets` not conform to the appropriate `Collection` and `Equatable` protocols? Why is `ChartDataEntry` [and ideally `ChartData` and `ChartDataSets`] a class and not a struct?) that could use enhancements to make it feel more like a Swift library.

I get the impression from the readme that this is supposed to be a community project. It's fine that the lead maintainer is away to be with their family. It's been a few months now without him though, so how long do we wait until we choose someone else to lead the project? Ideally more than one person so as to not fall in the same trap of having nobody qualified to make bigger decisions again? From reading responses to issues and PRs, it seems nobody currently reviewing them really has sufficient knowledge of Swift to make a decision on more fundamental changes to the framework (which will be required to handle larger datasets).

The frustrating part isn't that someone doesn't have time to look at a given PR, it's that even if they do nothing will happen. For example: this is a pretty simple PR that doesn't change the usage or performance of the framework, and is something that is required to make the framework "more Swifty" and (I can't say required) is a helpful step in making the framework be much more performant. With the exception of missing the case when both `lhs.data` and `rhs.data` are `nil`, there is no reason for it not to be accepted.